content/en/docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -637,13 +637,13 @@ Get the Pods to view their container images:
 for p in 0 1 2; do kubectl get pod "web-$p" --template '{{range $i, $c := .spec.containers}}{{$c.image}}{{end}}'; echo; done
 ```
 ```
-registry.k8s.io/nginx-slim:0.8
-registry.k8s.io/nginx-slim:0.8
-registry.k8s.io/nginx-slim:0.8
+gcr.io/google_containers/nginx-slim:0.8
```

Comment:
I don't believe we want to switch from `registry.k8s.io` to `gcr.io`. In general, we should be moving to project controlled services and not Google specific ones.

The patch command earlier was probably overlooked [when this migration was done](https://github.com/kubernetes/website/pull/35803), so I think it should be updated there rather than here.